python/whylogs/api/whylabs/session/config.py

```diff
@@ -3,24 +3,43 @@
 import logging
 import os
 from configparser import ConfigParser
+from dataclasses import dataclass
 from enum import Enum
 from pathlib import Path
 from typing import Optional
 
 from platformdirs import user_config_dir
 
 from whylogs.api.whylabs.session.notebook_check import is_notebook
+from whylogs.api.whylabs.session.prompts import (
+    prompt_api_key,
+    prompt_default_dataset_id,
+    prompt_org_id,
+    prompt_session_type,
+)
+from whylogs.api.whylabs.session.session_types import ApiKeyV1, ApiKeyV2
+from whylogs.api.whylabs.session.session_types import InteractiveLogger as il
+from whylogs.api.whylabs.session.session_types import (
+    SessionType,
+    parse_api_key,
+    parse_api_key_v2,
+)
 
 _DEFAULT_WHYLABS_HOST = "https://api.whylabsapp.com"
 _CONFIG_APP_NAME = "whylogs"
+_INIT_DOCS = "https://docs.whylabs.ai/docs/whylabs-whylogs-init"
```

Comment:
I'm currently working on a doc page for this new auth system on docs.whylabs.ai. I link to it in several places since the logic can be pretty verbose. It will be here when I'm done and we should hold off on merging until I finish `https://docs.whylabs.ai/docs/whylabs-whylogs-init`

Reply:
Feel free to comment on the url and side bar location 
![image](https://github.com/whylabs/whylogs/assets/1233709/3e557ada-1cc8-4edd-8872-8fa2d128f0ca)

Reply:
Draft of the new doc page: https://64b5bca8f6fdda0d2fff5639--whylabs-docs-dev.netlify.app/docs/whylabs-whylogs-init